rules/windows/image_load/image_load_susp_dll_load_system_process.yml

```diff
@@ -21,7 +21,10 @@ detection:
             - '\AppData\Local\Temp\'
             - 'C:\PerfLogs\'
     filter:
-        ImageLoaded|contains: '\Program Files'
+        ImageLoaded|startswith:
+            - 'C:\Program Files\'
+            - 'C:\Program Files (x86)\'
+            - 'C:\ProgramData\'
```

Comment:
I wouldn't add the ProgramData folder since normal users can create folders in it.